internal/docker/auth.go

```diff
@@ -2,50 +2,42 @@ package docker
 
```

Comment:
Noticed this package doesn't have documentation, since you're working on this might as well put docs on it

Reply:
@shuminyang it does have package docs in the docker.go file